src/VCR/Util/SoapClient.php

```diff
@@ -2,13 +2,14 @@
 
 namespace VCR\Util;
 
+use SoapClient as BaseSoapClient;
 use VCR\LibraryHooks\SoapHook;
 use VCR\VCRFactory;
 
 /**
  * SoapClient replaces PHPs \SoapClient to allow interception.
  */
-class SoapClient extends \SoapClient
```

Comment:
Do you know why this could be only an issue when running under windows? Is there a difference?